File: Örnek-6.5/Java/kenaruzun.java
package algoritma;
import java.util.Scanner;

public class kenaruzun {

	public static void main(String[] args) {
		double a,b,c,açı;
		Scanner scan=new Scanner(System.in);
		
		System.out.print("Birinci kenar uzunluğunu giriniz (cm): ");
		a=scan.nextDouble();
		
		System.out.print("İknci kenar uzunluğunu giriniz (cm): ");
		b=scan.nextDouble();
		
		System.out.print("Arasındaki açıyı giriniz (derece): ");
		açı=scan.nextDouble();
		
		c=Math.sqrt(a*a+b*b-2*a*b*Math.cos(Math.toRadians(açı)));
		
		System.out.println("C kenarın uzunluğu (cm): "+c);
		
	}

}

File: Örnek-6.5/C++/kenaruzun.cpp
#include <iostream>
#include <math.h>
using namespace std;

int main() {
	double a,b,c,aci;
	cout<<"Birinci kenar uzunlugunu giriniz (cm): ";
	cin>>a;
	cout<<"Ikinci kenar uzunlugunu giriniz (cm): ";
	cin>>b;
	cout<<"Arasindaki aciyi (derece): ";
	cin>>aci;
	
	c=sqrt(a*a+b*b-2*a*b*cos(M_PI*aci/180));
	
	cout<<"C kenarin uzunlugu: "<<c;
	
	return 0;
}
